### DISCUSSION TITLE
Who is interested in working on CubeDesk?

### DISCUSSION THREAD

**Discussion by kash** [kash/cubedesk#101] [Q&A]
Hi all! Now that CubeDesk is open source, I just wanted to gauge how many people would be interested in working on the code. This can be something as simple as making minor design changes to being really involved and making major infrastructure changes. 

If you're interested, leave a reply below! And please mention what sort of things you'd like to work on.

**Comment by JupiLogy** [comment]
Hi! I'd be interested in helping out. I have coding experience on projects though not normally stuff with a large number of people actually using it.

Front end design isn't my forte but I am interested in ironing out sessions/stats/scrambles/sharing, whatever "functionality" stuff really!

Most of my GitHub projects are a bit old as I'm still doing coding, but on private projects/stuff outside of GitHub.

Really excited that cubedesk is open source btw!

Thanks,
Jupy

**Comment by Wo0fle** [comment]
Hello!

First of all, I would like to say that it is awesome to see such a great project like this be open-source.

I have some experience in web development, but I'm still pretty new to the field and I'm learning as I go along.

I would love to be able to help in any way I can!

**Reply by kash** [reply]
That's great to hear! Would love to have you help out. I have some instructions in the Wiki on how to get the repo set up. Let me know if you have any issues with it. Once you have it set up, let me know if you have some ideas of things you'd like to contribute.

**Comment by Kyo4891** [comment]
Hi there,

I'm keen to help out if I can. Specifically around the bluetooth / smartcube aspect of it. I used to be a full time dev for C#.net but have since changed roles. But this sounds like a fun project to get myself back into coding.

Cheers.

**Reply by kash** [reply]
Hey! Really great to hear as that's one of the lacking parts of CubeDesk. There are setup instructions in the Wiki but please let me know if you run into issues. Happy to share potential tasks/projects once you're set up.

**Reply by Kyo4891** [reply]
Has anyone tried to set up the dev environment using Ubuntu in Hyper-V?
I ran into some issues with the postgres part. Initially with creating the user (I tried a different way which worked) and later on initiating the DB.

Update: managed to get it working in Hyper-V but that doesnt allow Bluetooth / hardware passthrough which is an issues since I wanna look into the smartcube side of the code.  Got it working in native Ubuntu as well.  The only thing I needed to improvise was the Node, I needed to download an older compatible version. Also I had to find other ways to create the user and DB, again no biggie
